flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCInputFormat.java

```diff
@@ -116,6 +116,7 @@
 	private transient PreparedStatement statement;
 	private transient ResultSet resultSet;
 	private int fetchSize;
+	private Boolean autoCommit;
```

Comment:
I'd add a comment to explain that we use `Boolean` instead of `boolean` to respect the default settings of the driver (even if it violates the JDBC standard which requires auto commit to be `true` by default).